salt/haproxy/haproxy.cfg.jinja

```diff
@@ -1,4 +1,4 @@
-{%- if "kube-master" in salt['grains.get']('roles', []) -%}
+{%- if "admin" or "kube-master" in salt['grains.get']('roles', []) -%}
```

Comment:
I'm not sure this is needed / wanted? We don't want people using the admin node address to access K8S, or Dex, directly - as it's a SPOF.

Reply:
I agree

Reply:
Updated